packages/template-retail-react-app/app/pages/account/payments.test.js

```diff
@@ -22,21 +26,61 @@ jest.mock('@chakra-ui/toast', () => {
 })
 
 const MockedComponent = () => {
-    const customer = useCustomer()
+    const {isRegistered} = useCustomerType()
+    const login = useShopperLoginHelper(ShopperLoginHelpers.LoginRegisteredUserB2C)
+    const customer = useCurrentCustomer()
     useEffect(() => {
-        customer.login('test@test.com', 'password')
+        if (!isRegistered) {
+            login.mutate(
+                {email: 'email@test.com', password: 'password1'},
+                {
+                    onSuccess: () => {
+                        window.history.pushState({}, 'Account', createPathWithDefaults('/account'))
+                    }
+                }
+            )
+        }
```

Comment:
I suppose we don't need to do anything in `onError` or `onSettled` because we didn't before? Just double checking